IdentityCore/src/util/NSString+MSIDExtensions.h

```diff
@@ -40,6 +40,9 @@
 /*! Returns the same string, but without the leading and trailing whitespace */
 - (NSString *)msidTrimmedString;
 
+ /*! Returns the string lowercased, but without the leading and trailing whitespace */
+- (NSString *)msidNormalizedString;
```

Comment:
you don't seem to use it in prod?

Reply:
It's being used up in MSAIStorageManagerImpl.m, which is outside the msalobjc repo. But it should probably be moved down into a msalobjc-based utility. I'll look at it further.

Reply:
I've eliminated where I was using outside of test code, so I'll cut it entirely, just doing it the way msalobjc does it, like `environment.msidTrimmedString.lowercaseString`.